strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/validation/groupId/GroupIdValidator.java

```diff
@@ -0,0 +1,26 @@
+package org.carlspring.strongbox.storage.validation.groupId;
+
+import org.carlspring.strongbox.providers.ProviderImplementationException;
+import org.carlspring.strongbox.providers.io.RepositoryPath;
+
+import java.io.IOException;
+
+/**
+ * Created by dinesh on 12/6/17.
+ */
+public interface GroupIdValidator {
```

Comment:
Similarly to the exception comment above, I think this could be `LowercaseValidator`...